editor/editor_node.cpp

```diff
@@ -4330,9 +4330,19 @@ Ref<Texture2D> EditorNode::_get_class_or_script_icon(const String &p_class, cons
 			return gui_base->get_theme_icon(p_class, SNAME("EditorIcons"));
 		}
 
-		if (p_fallback.length() && gui_base->has_theme_icon(p_fallback, SNAME("EditorIcons"))) {
+		if (!p_fallback.is_empty() && gui_base->has_theme_icon(p_fallback, SNAME("EditorIcons"))) {
 			return gui_base->get_theme_icon(p_fallback, SNAME("EditorIcons"));
 		}
+
+		// If the fallback is empty or wasn't found, use the default fallback.
+		if (ClassDB::class_exists(p_class)) {
+			bool instantiable = ClassDB::is_virtual(p_class) || !ClassDB::can_instantiate(p_class);
```

Comment:
This is confusingly named

Reply:
Why and how should I rename it?

Reply:
The boolean is true when the class *isn't* instantiable, I'd either invert the boolean and swap the values below, or rename it to `non_instantiable`